app/views/Project/Tagging/Sources/SourcesFilter/EntryFilter/FrameworkFilterItem/index.tsx

```diff
@@ -0,0 +1,389 @@
+                        && styles.hidden,
+                    )}
+                    name="valueList"
+                    value={value?.valueList}
+                    onChange={onFieldChange}
+                    label={title}
+                    options={filter.properties.options}
+                    keySelector={filterKeySelector}
+                    labelSelector={filterLabelSelector}
+                    disabled={disabled || optionsDisabled}
+                />
+            );
+        }
+        case 'GEO': {
+            return (
+                <GeoMultiSelectInput
+                    name="valueList"
+                    className={_cs(
+                        className,
+                        styles.input,
+                        (hasNoData(value?.valueList) && !allFiltersVisible)
+                        && styles.hidden,
+                    )}
+                    value={value?.valueList}
+                    onChange={onFieldChange}
+                    label={title}
+                    projectId={projectId}
+                    options={geoAreaOptions}
+                    onOptionsChange={setGeoAreaOptions}
+                    disabled={disabled}
+                    placeholder={title}
+                />
+            );
+        }
+        case 'SELECT': {
+            return (
+                <SelectInput
+                    className={_cs(
+                        className,
+                        styles.input,
+                        (hasNoData(value?.value) && !allFiltersVisible)
+                        && styles.hidden,
+                    )}
+                    name="value"
+                    value={value?.value}
+                    label={title}
+                    onChange={onFieldChange}
+                    options={filter.properties.options}
+                    keySelector={filterClientIdSelector}
+                    labelSelector={filterLabelSelector}
+                    disabled={disabled || optionsDisabled}
+                />
+            );
+        }
+        case 'MULTISELECT': {
+            return (
+                <MultiSelectInput
+                    className={_cs(
+                        className,
+                        styles.input,
+                        (hasNoData(value?.valueList) && !allFiltersVisible)
+                        && styles.hidden,
+                    )}
+                    name="valueList"
+                    value={value?.valueList}
+                    onChange={onFieldChange}
+                    label={title}
+                    options={filter.properties.options}
+                    keySelector={filterClientIdSelector}
+                    labelSelector={filterLabelSelector}
+                    placeholder={title}
+                    disabled={disabled || optionsDisabled}
+                />
+            );
+        }
+        case 'ORGANIGRAM': { // FIXME options sent by the server is gibberish (null values)
```

Comment:
@thenav56 we may have changed the data structure for organigram so the filterable data may not be correct for organigram.

Reply:
Current structure used in backend (from new branch). https://github.com/the-deep/server/blob/ebe7e51b91f08832ee17d54a0a5b5d12c0c85e09/apps/analysis_framework/widgets/organigram_widget.py#L3-L11.

Let me know if this still persists.

Reply:
Let me quicky verify if this is the case for me as well